src/wp-includes/taxonomy.php

```diff
@@ -1559,6 +1559,18 @@ function term_exists( $term, $taxonomy = '', $parent = null ) {
 		$defaults['fields']   = 'all';
 	}
 
+	/**
+	 * Filter default arguments for checking if term exists.
```

Comment:
Use third-person verbs for better consistency with other docblocks (= "Filters …")

Reply:
Better?